src/main/webapp/resources/js/uk.js

```diff
@@ -1 +1,137 @@
\ No newline at end of file
+!function (a) {
+    "function" == typeof define && define.amd ? define(["jquery", "moment"], a) : "object" == typeof exports ? module.exports = a(require("jquery"), require("moment")) : a(jQuery, moment)
+}(function (a, b) {
+    !function () {
+        "use strict";
+        function a(a, b) {
+            var c = a.split("_");
+            return b % 10 === 1 && b % 100 !== 11 ? c[0] : b % 10 >= 2 && 4 >= b % 10 && (10 > b % 100 || b % 100 >= 20) ? c[1] : c[2]
+        }
+
+        function c(b, c, d) {
```

Comment:
you should name your functions and parameters with more meaningful names
c - is bad name for function
b, c, d, a - bad names for parameters

good name should give you information about function or parameter from name only, not even reading comment or javadoc.